### ISSUE TITLE
Wrong version of poppler executable detected

### ISSUE THREAD

**Issue by nodje** [Belval/pdf2image#171]
pdf2image.py:425
`err.decode("utf8", "ignore").split("\n")[0].split(" ")[-1].split(".")[1]`
returns `10` for 
`err =b'pdftocairo version 20.10.0\nCopyright 2005-2020 The Poppler Developers - http://poppler.freedesktop.org\nCopyright 1996-2011 Glyph & Cog, LLC\n'` 
On Macos

**Comment by Belval** [comment]
This is pretty bad. I will change how the version is detected. Thank you for reporting it.

**Comment by ooojpeg** [comment]
Hi @Belval I just had a quick look at this and just wanted to share an easy fix(that i'm sure you know) and one that is more robust.

**Simple**
`err.decode("utf8", "ignore").split("\n")[0].split(" ")[-1]`

**More robust**
`re.findall("pdftocairo\sversion\s\d[0-100].\d[0-99].\d", err.decode("utf8", "ignore"))[0].split(" ")[2]`

If you would like me to submit a request just let me know what one you prefer. :)

**Comment by Belval** [comment]
Yes actually, I haven't had time to look at this issue so if you can make a PR I'd appreciate it.

**Comment by ooojpeg** [comment]
@Belval Not sure if you had time to look at my PR or if you think anything needs changed. Thanks.
